infra/token-transfer-server/src/config.js

```diff
@@ -26,7 +26,9 @@ if (!sendgridFromEmail) {
   process.exit(1)
 }
 
-const unlockDate = moment.utc('2020-01-01')
+const unlockDate = process.env.UNLOCK_DATE
```

Comment:
Wonder if it's dangerous to have a default in case we somehow misconfigure EnvKey. Should we rather crash if UNLOCK_DATE is not defined ?

Reply:
👍